src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CngKey.EC.cs

```diff
@@ -115,14 +112,17 @@ internal static CngAlgorithm EcdhCurveNameToAlgorithm(string name)
             {
                 case "nistP256":
                 case "ECDH_P256":
+                case "ECDSA_P256":
```

Comment:
There were two copies of these, one had this line, the other didn't.  Apparently one got hit by ECDiffieHellman.Create(), the other git hit by ECDiffieHellmanCng.